src/saga/adaptors/lsf/lsfjob.py

```diff
@@ -198,6 +203,14 @@ def _lsfcript_generator(url, logger, jd, ppn, lsf_version, queue, span):
 
     lsf_params += "#BSUB -n %s \n" % str(jd.total_cpu_count)
 
+    if jd.total_node_count is None:
+        jd.total_node_count = 1
+
+    lsf_params += "#BSUB -nnodes %s \n" %str(jd.total_node_count)
+
+    if jd.alloc_flags is not None:
+        lsf_params += "#BSUB -alloc_flags %s \n" % ', '.join(jd.alloc_flags)
+
```

Comment:
Can you please explain what the semantics of this alloc flag is on summit?

Reply:
https://www.olcf.ornl.gov/for-users/system-user-guides/summit/running-jobs/#common-bsub-options

Reply:
Should this be:
```
lsf_params += '#BSUB -alloc_flags "%s" \n' % ' '.join(jd.alloc_flags)
```

Note the quotation and separation by space above. The example in the referenced url specifies the flag as:
```
#BSUB -alloc_flags "gpumps smt1"
```

Reply:
You are right. For some reason I saw a comma